Berksfile

```diff
@@ -0,0 +1,18 @@
+source 'https://supermarket.chef.io'
+
+cookbook 'base',
+         git: 'git@github.com:osuosl-cookbooks/base'
+
```

Comment:
Its fine with the intent, but generally we haven't been making a new line space between each cookbook line after that.